1907-07-01.xml

```diff
@@ -1170,11 +1170,404 @@
                             <!-- No template for my newspaper's version of THE Egyptian State Railways -->
                 </div>
             <div type="page" n="2"> </div>
```

Comment:
Move the `</div>` tag, which ends page 2, to the end of page 2, i.e. just before page 3 starts.